addons-l10n/en/blocks2image.json

```diff
@@ -3,5 +3,7 @@
   "blocks2image/export_all_to_SVG": "Export all as SVG",
   "blocks2image/export_selected_to_PNG": "Export block as PNG",
   "blocks2image/export_all_to_PNG": "Export all as PNG",
-  "blocks2image/error_blocks_not_added": "Add blocks to workspace!"
+  "blocks2image/error_blocks_not_added": "Add blocks to workspace!",
```

Comment:
This really doesn't matter but this at the bottom to keep all the context menu strings together.